lib/fastly_nsq/feeder.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class FastlyNsq::Feeder
+  attr_reader :processor, :priority
+
+  def initialize(processor, priority)
+    @processor = processor
+    @priority = priority
+  end
+
+  # @see http://ruby-concurrency.github.io/concurrent-ruby/Concurrent/ThreadPoolExecutor.html#post-instance_method
+  # @see {Nsq::Connection#read_loop}
+  def push(message)
+    FastlyNsq.manager.pool.post(priority) { processor.call(message) }
```

Comment:
processor here would be the class consuming the messages?  If so, any reason to not just leave the existing convention of a method called `process` per processor?

Reply:
`processor` here is the instance of `FastlyNsq::Listener`.  I had made an 'executive' decision to switch to '#call` so everything is compatible with blocks.  Completely open for discussion but I like the flexibility.

Reply:
I agree about the use of `call` moving forward, but at the least I think we should support an alias to process while we're rolling this out.

Reply:
I was comfortable changing this now because of the major version bump that is required due to the other changes.  If you feel strongly about it, an alias will definitely work.

Reply:
We are really the only users currently of this... with a major bump we are going to have to run around and change a bunch of stuff wherever this is used already. Mine as well make the change now so we can do it all at once!

Reply:
good points; sold.  I do like that we don't have to change the classes and can just change the configuration to have it call .process in a proc.